src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,19 @@
+import "./Navbar.css";
+import { NavLink } from "react-router-dom";
+
+function Navbar() {
+  return (
+    <div className="navbarContainer">
+      <nav>
+        <button className="baby-btn">
```

Comment:
You've made a button component before, so use that component here. We want to reuse our reusable components. No need to make buttons all over again.